synapse/config/registration.py

```diff
@@ -270,19 +282,30 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
         #  - matrix.org
         #  - vector.im
 
-        # Handle threepid (email/phone etc) registration and password resets
-        # through a *trusted* identity server. Note that this allows the configured
-        # identity server to reset passwords for accounts.
+        # Handle threepid (email/phone etc) registration and password resets through a set of
+        # *trusted* identity servers. Note that this allows the configured identity server to
+        # reset passwords for accounts!
+        #
+        # Also be aware that if email is not set to a domain, and SMTP options have not been
```

Comment:
> email is not set to a domain

I don't really understand what this means?

Reply:
oh, do you just mean:

```suggestion
        # Be aware that if `email` is not set, and SMTP options have not been
```

Reply:
yep